_data/docsarchive/build_all_archives.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+ARCHIVE_BRANCHES=("v1.4" "v1.5" "v1.6" "v1.7" "v1.8" "v1.9" "v1.10" "v1.11" "master")
```

Comment:
eventually, this won't be needed if each branch has a Dockerfile in it, then we can setup auto builds for each branch